android/src/main/java/com/thebluealliance/androidclient/activities/settings/NotificationSettingsActivity.java

```diff
@@ -51,18 +52,19 @@ public void onCreate(Bundle savedInstanceState) {
             notificationTone.setEnabled(currentlyEnabled);
             notificationVibrate.setEnabled(currentlyEnabled);
 
-            enableNotifications.setOnPreferenceChangeListener(new Preference.OnPreferenceChangeListener() {
-                @Override
-                public boolean onPreferenceChange(Preference preference, Object newValue) {
-                    boolean value = (Boolean) newValue;
-                    notificationTone.setEnabled(value);
-                    notificationVibrate.setEnabled(value);
-                    if (Build.VERSION.SDK_INT > Build.VERSION_CODES.KITKAT) {
+            enableNotifications.setOnPreferenceChangeListener((preference, newValue) -> {
+                boolean value = (Boolean) newValue;
+                notificationTone.setEnabled(value);
+                notificationVibrate.setEnabled(value);
+                if (Build.VERSION.SDK_INT > Build.VERSION_CODES.KITKAT) {
```

Comment:
Could probably take out this `if`; the null checks will take care of the case where these preferences don't exist on Lollipop devices